src/gsi/read_guess.F90

```diff
@@ -164,8 +167,28 @@ subroutine read_guess(iyear,month,idd,mype)
         else if (nems_nmmb_regional) then
            call nmm_binary_guess%read_nems_nmmb_guess(mype)
         else if (fv3_regional      ) then
-           call bg_fv3regfilenameg%init
-           call  read_fv3_netcdf_guess(bg_fv3regfilenameg)
+           it_c=0
+           do it=1,7 !Xu default only -+3 hours for FGAT
+              if(it.eq.4) then
```

Comment:
I suggest to put the nfldsig loop of reading ( for it) into the read_fv3_netcdfguess subroutine, as does regional wrf does. 
Another problem is about the naming convention of fv3reg files on different time levels.  Can we make it be compatible to the "old runs" or we need a new convention?  like we need to set the name of bg files on the analysis time : fv3_dynvar_01 (for hourly da ,and so on)?

Reply:
Thanks for the suggestion. The read_guess has been modified accordingly using the nfldsig.  
But the bg files was named as fv3_dynvars_03~fv3_dynvars_09, not sure about the last question.